lua/ttt2/libraries/targetid.lua

```diff
@@ -0,0 +1,558 @@
+---
+-- @module TargetID
+-- @desc Handling operations to display the TargetID
+-- @author Mineotopia
```

Comment:
maybe you want to add yourself here too?

Reply:
Wasn't sure if I'm an author of this library as I just reassembled code, that was there ^^ Wouldn't mind adding me there.

Reply:
Feel free to add yourself!